dev-riscv/tests/T1.jasm

```diff
@@ -0,0 +1,37 @@
+package  java/lang;
```

Comment:
Can we assemble `java/lang/Object.class`? We'll avoid introducing new classes (JDK) and new class names (hotspot)

Reply:
My intention was to populated new tests by some kind of regression test until we are able to exploit ordinary openjdk test suites.